cores/esp32/esp32-hal-gpio.c

```diff
@@ -244,6 +253,10 @@ extern void __attachInterrupt(uint8_t pin, voidFuncPtr userFunc, int intr_type)
     esp_intr_enable(gpio_intr_handle);
 }
 
+extern void __attachInterrupt(uint8_t pin, voidFuncPtr userFunc, int intr_type) {
+	__attachInterruptArg(pin, (voidFuncPtrArg)userFunc, NULL, intr_type);
```

Comment:
The casting (voidFuncPtrArg) looks redundant.

Reply:
yes, but then I get a compiler warning.